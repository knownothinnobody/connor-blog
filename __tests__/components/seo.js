import React from "react"
import * as Gatsby from "gatsby"
import { render, shallow, mount } from "enzyme"

import SEO from "../../src/components/seo"

const useStaticQuery = jest.spyOn(Gatsby, "useStaticQuery")

beforeEach(() => {
  useStaticQuery.mockImplementation(() => ({
    site: {
      siteMetadata: {
        title: "Site Title",
        description: "Site Description",
        author: "Author",
      },
    },
  }))
})

describe("SEO", () => {
  it("renders correctly", () => {
    expect(render(<SEO title="Page Title" />)).toMatchSnapshot()
  })
})
