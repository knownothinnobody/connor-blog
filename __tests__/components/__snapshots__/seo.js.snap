// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO renders correctly 1`] = `
Object {
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
